nvflare/private/fed/app/client/worker_process.py

```diff
@@ -154,6 +154,14 @@ def parse_arguments():
     parser.add_argument("--sp_target", "-g", type=str, help="Sp target", required=True)
     parser.add_argument("--sp_scheme", "-scheme", type=str, help="Sp connection scheme", required=True)
     parser.add_argument("--parent_url", "-p", type=str, help="parent_url", required=True)
+    parser.add_argument(
```

Comment:
The convention should be to use -- for word options, - for single letter ones. We already broke it.

Reply:
It's been like this, no need to change now.